src/lang/pt-br.json

```diff
@@ -184,25 +189,25 @@
 	"MOVEMENT": "Movimento",
 	"MODIFIERS": "Modificadores",
 	"NOTIFY.YOU_ARE_BROKEN": "Você está acabado!",
-	"NOTIFY.YOUR_ITEM_BROKE": "Seu equipamento está quebrado!",
+	"NOTIFY.YOUR_ITEM_BROKE": "Seu item está quebrado!",
 	"OTHER": "OUTROS",
 	"POWER_LEVEL": "Nível de Poder",
 	"PUSH": "FORÇAR!",
-	"PUSHED": "Forçou",
+	"PUSHED": "Forçada",
 	"RANGE.ARM": "Ao Alcance das Mãos",
 	"RANGE.DISTANT": "Distante",
 	"RANGE.LONG": "Longa",
 	"RANGE.NEAR": "Perto",
 	"RANGE.SHORT": "Curta",
-	"RAW_MATERIAL.SHELF_LIFE": "Vida Útil",
-	"ROLL.APPLIED_TRAUMA": "Dano aplicado em",
-	"ROLL.CHANCE": "Lançamento Arriscado: Role um infortúnio mágico.",
-	"ROLL.MISHAP": "Role um infortúnio mágico!",
-	"ROLL_MODIFIER.SELECT": "Selecione um modificador",
+	"RAW_MATERIAL.SHELF_LIFE": "Tempo Limite de Estocagem",
```

Comment:
There are many words here that are long, it is necessary to see how they are presented in the frontend

Reply:
This one I wasn't a bit sure if it would fit, but it more "book right" than "Vida útil". If it is too long I would choose "Validade" as it has a more clear meaning.

Reply:
Just checked, that is indeed too long, I would recommend "Validade" here instead.